README.md

```diff
@@ -44,45 +70,101 @@ The structure should appear like so:
 \---specification
 |    +---automation
 |    |   \---resource-manager
-|    |       \---Microsoft.Automation
-|    |           \---stable
-|    |               \---2015-10-31
-|    |                   \---examples
+|    |       +---Microsoft.Automation
+|    |       |   \---stable
+|    |       |       \---2015-10-31
+|    |       |           \---examples
+|    |       \---readme.md
 |    +---batch
 |    |   +---data-plane
-|    |   |   \---Microsoft.Batch
-|    |   |       +---stable
-|    |   |       |   +---2015-12-01.2.2
-|    |   |       |   +---2016-02-01.3.0
-|    |   |       |   +---2016-07-01.3.1
-|    |   |       |   +---2017-01-01.4.0
-|    |   |       |       \---examples
-|    |   |       \---preview
-|    |   |           \---2017-05-01.5.0
+|    |   |   +---stable
+|    |   |   |   +---2016-07-01
+|    |   |   |   |   \---examples
+|    |   |   |   \---2017-01-01
+|    |   |   |       \---examples
+|    |   |   +---preview
+|    |   |   |   \---2017-05-01-preview
+|    |   |   |       \---examples
+|    |   |   \---readme.md
 |    |   \---resource-manager
-|    |       \---Microsoft.Batch
-|    |           +---stable
-|    |           |   +---2015-12-01
-|    |           |   +---2017-01-01
-|    |           |       \---examples
-|    |           \---2017-05-01
-|    |               \---examples
-|    +---billing
+|    |       +---Microsoft.Batch
+|    |       |   \---stable
+|    |       |       +---2015-12-01
+|    |       |       |   \---examples
+|    |       |       +---2017-01-01
+|    |       |       |   \---examples
+|    |       |       \---2017-05-01
+|    |       |           \---examples
+|    |       \---readme.md
+|    +---playfab
 |        \---resource-manager
-|            \---Microsoft.Billing
-|                \---stable
-|                |   +---2017-02-27-preview
-|                |       \---examples
-|                +---preview
-|                    \---2017-04-24-preview
-|                        \---examples
-\--- readme.md
+|            +---cadl
+|            +---Microsoft.Playfab
+|            |   +---stable
+|            |   |   \---2017-02-27-preview
+|            |   |       \---examples
+|            |   \---preview
+|            |       \---2017-04-24-preview
+|            |           \---examples
+|            \---readme.md
+```
+### Folder Structure for Service Group
+If several services with the same or different versioning cycle decide to package together in the SDK release. For the service group, the folder structure is recommended as below:
+
+```bash
+.
+\---specification
```

Comment:
based on our discussion, here will be changed to use proposal 2, sub folder is under resource-manager/<namespace>